src/styles.css

```diff
@@ -1,2 +1,28 @@
 /* You can add global styles to this file, and also import other style files */
 @import "~bootstrap/dist/css/bootstrap.css";
+@import "~@ng-select/ng-select/themes/default.theme.css";
+
+.btn-primary {
+  color: #212529;
+  background-color: #ff8109;
+  border-color: #ff8109;
+}
```

Comment:
This is fine but we probably don't want to go too overboard with customizations, implementors could/will want to apply their own style